test.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+chomp() {
+    echo -n "${@//[[:space:]]/}"
+}
+
+readonly EXPECTED_NODE_VERSION="$@"
+readonly NODE_VERSION="$(chomp $(./node --version))"
```

Comment:
For some crazy reason node also prints out the version number suffixed with `\r`.